src/clarity-angular/data/datagrid/providers/selection.ts

```diff
@@ -85,6 +86,7 @@ export class Selection {
     private get _selectable(): boolean {
         return (this._selectionType === SelectionType.Multi) || (this._selectionType === SelectionType.Single);
     }
+
```

Comment:
Why do we have formatting changes even with clang-format? Is it still properly working?

Reply:
@Shijir - are you running the pre-commit hook or `npm run clang:format` before you commit changes? 
This is the pre-commit hook I'm currently using: https://gist.github.com/hippee-lee/5d3e83f02d90224abb5de9e50893deca

Reply:
Sorry about this, I usually add an empty line after methods to make it more readable. I do use that `clang:format` on every commit, but looks like it doens't remove them. I have manually removed these empty lines.

Reply:
Thanks for clarifying. The default is for clang:format to remove them but it seems something changed with the change to webpack. I'll investigate further.